front/src/routes/integration/all/nextcloud-talk/NextcloudTalk.jsx

```diff
@@ -1,15 +1,42 @@
 import { Text, MarkupText, Localizer } from 'preact-i18n';
 import cx from 'classnames';
 import { RequestStatus } from '../../../../utils/consts';
+import DeviceConfigurationLink from '../../../../components/documentation/DeviceConfigurationLink';
 
 const NextcloudTalkPage = ({ children, ...props }) => (
   <div class="page">
     <div class="page-main">
       <div class="my-3 my-md-5">
         <div class="container">
           <div class="row">
-            <div class="col-lg-12">
+            <div class="col-lg-3">
+              <h3 class="page-title mb-5">
+                <Text id="integration.nextcloudTalk.title" />
+              </h3>
+              <div>
+                <div class="list-group list-group-transparent mb-0">
+                  <DeviceConfigurationLink
+                    user={props.user}
+                    configurationKey="integrations"
+                    documentKey="nextcloudTalk"
```

Comment:
This doesn't link to any documentation ! 

Maybe, let's put "nextcloud-talk" as the key and add a new documentation on the website?

Reply:
Both documentation are live!